bindings/go/oci/looseref/parse_test.go

```diff
@@ -119,17 +122,43 @@ func TestLooseParseReference(t *testing.T) {
 				Tag: "v1.0.0",
 			},
 		},
+		{
```

Comment:
I would like to see some error cases here:
- tagging with an invalid digest
- partial digests or malformed digest strings